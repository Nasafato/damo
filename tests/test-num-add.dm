def test() : int {
	// Variable declarations come first
	num i;
	num j;

	// Statements come second
	j = 2.0;
	i = 1.0;
	print_num(i+j);
	return 0;
}

test();