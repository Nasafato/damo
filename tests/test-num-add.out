0.400000
1.700000
2.100000